src/components/CommentAuthorAvatar.jsx

```diff
@@ -24,6 +24,12 @@ class CommentAuthorAvatar extends React.Component {
         this._ismounted = true;
         fetchUserName(this.props.accessToken, this.props.userId)
             .then((responseJson) => {
+                if (!responseJson || !responseJson.profile) {
```

Comment:
What would happen in the 'esle' case?

Reply:
It's right below:

```
this.safeSetState({
                    avatar: responseJson.profile.picture,
                    name: responseJson.profile.name,
                });
```
runs only if the profile is present.

Reply:
I think I placed the question in the wrong place. The other two places do not seem to have `else`.